build.sbt

```diff
@@ -80,3 +94,21 @@ fork in Test := true
 
 // forward stdin/out to fork, so the OpenIE CLI can be run in sbt.
 connectInput in run := true
+
+// The OpenIE codebase has many insignificant style warnings that appear
+// whenever the code is compiled. There are also some important-sounding
+// warnings which don't seem to affect the behavior but should be
+// fixed eventually:
+//
+//   Classes must override both equals and hashCode, not just one
+//   Line is more than 100 characters long
+//   Tab in Scala file
+//   Unusual class name; consider renaming
+//   illegal.imports.message
```

Comment:
I wonder if we can be more granular with ignoring these?  A discrepancy between `equals` and `hashCode` feels like it could produce legitimate problems, so we might not want to hide that warning.  "Tab in Scala file" also seems like it'd be easy to fix with a global find-and-replace in IntelliJ.

Our SBT plugins already expose [a setting](https://github.com/allenai/sbt-plugins/blob/master/src/main/scala/org/allenai/plugins/StylePlugin.scala#L24) to globally enable / disable checking line length.  I think it'd be better to follow that pattern and explicitly pick out the style checks we want to disable rather than turn off everything by default.